### ISSUE TITLE
difference to Jitsi?

### ISSUE THREAD

**Issue by aspiers** [jangouts/jangouts#154]
On the surface, Jangouts and [Jitsi](https://jitsi.org/) appear to have very similar goals - to me, at least. It would be helpful if there was an explanation of the difference between the two in an easy to find location, e.g. in the README, or in a FAQ linked from the README.  Ideally this would answer:

- What are the current differences to the end user functionality?
- What are the differences in the projects' goals / missions?
- What are the most important differences in the technical implementation?

**Comment by aspiers** [comment]
In answer to @bmwiedemann's [comment in #153](https://github.com/jangouts/jangouts/issues/153#issuecomment-244013644), yes, I tried Jitsi and I was absolutely amazed how good it is.

**Comment by ancorgs** [comment]
Of course we tried all the available self-hosted solutions we were aware of before writing Jangouts. Our fault was not writing a report about it... or maybe such report exist as comments to the hackweek project? I'll check.

The problem with Jitsi is that it didn't scale at all with the number of participants in our tests. We usually have meetings with around 15 people in the room. Jitsi performed worse with every new participant in the room. It became totally unusable when the 6th or 7th participant joined.

**Comment by ancorgs** [comment]
No, we wrote no report. From the top of my memory we evaluated Jitsy, Talk.io, Telepresence, Palava, BigBlueButton, OpenMeetings, some demo client for Licode or Kurento (not sure which) and probably some more. None of them did fit our needs at that point in time.

**Comment by imobachgs** [comment]
Jitsy looked like the most appealing options to me but, as @ancorgs said, we had severe performance problems. Maybe we should check it again in a near future. Volunteers? :)

**Comment by aspiers** [comment]
I would have thought that fixing the scalability issues would have been easier than rewriting the whole thing from scratch, but maybe I'm missing something, e.g. does Jitsi have fundamental scalability limits with its architecture?

**Comment by ancorgs** [comment]
Well, it was Hack Week, you know. :wink:

**Comment by aspiers** [comment]
Hahah ;-)

**Comment by kiview** [comment]
We've had the same performance problems with Jitsi btw.

**Comment by imobachgs** [comment]
Maybe writing some documentation about how Jangouts compares to other solutions would be nice, but I do not expect to have any volunteer to write such a document for the time being.

**Comment by ancorgs** [comment]
I was talking to Jitsi developers recently. They have 12 people working full time in the project. That is 12 more people than Jangouts 😉.  And they told me they have invested quite some effort in the last couple of years to improve congestion control.

So maybe Jitsi is scalable enough nowadays and the main (only?) Jangouts advantage over Jitsi is gone.

**Comment by eroy4u** [comment]
So as of now is jangouts scalable? Like can it support 15 people doing video conference and beyond?
I'm also evaluating jangouts and jitsi and see which one is scalable.

**Comment by imobachgs** [comment]
@eroy4u We use Jangouts with around 15 people everyday and with no problems at all. It would be nice if you could share your findings :)

Thanks!

**Comment by joudinet** [comment]
This discussion is more than a year old but I've just discovered Jangouts and I can't use Jitsi due to its requirement on Java on the server (my server is an embedded device with no JVM). So a big advantage for Jangouts over Jitsi is the light requirements with regards to server deployment. It only needs a Janus server (a C project) and a web server to serve static files.
